addusers.py

```diff
@@ -257,6 +246,30 @@ def parse_rows(rows):
     return projects, bad_rows
 
 if __name__ == "__main__":
+    
+    help_description = ("Add new users and projects to OpenStack using " 
+                        "data from Google Sheets.")
+    parser = argparse.ArgumentParser(description=help_description)
+    parser.add_argument('-c', '--config', 
+                        help='Specify configuration file.')
+
+    args = parser.parse_args()
+   
+    if args.config is not None:
+        CONFIG_FILE = set_config_file(args.config)
+    else:
+        CONFIG_FILE = set_config_file()
+
+    config = ConfigParser.ConfigParser()
+    config.read(CONFIG_FILE)
+
+    admin_user = config.get('auth', 'admin_user')
+    admin_pwd = config.get('auth', 'admin_pwd')
```

Comment:
I don't think there's a simple way around that.  In production the scripts don't authenticate as the actual "admin" user - they use a different user with admin privileges.  Not much better but at least we can disable it if needed without breaking things.

I expect we could lock it down further by creating a Keystone role with permissions for only the API calls the script needs.  But that would take some time and doesn't fix the password in a file problem. 

I've set things up in the helpdesk VM so the helpdesk users won't be able to see this file.